automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainBase.scala

```diff
@@ -0,0 +1,84 @@
+package org.broadinstitute.dsde.firecloud.test.security
+
+import org.broadinstitute.dsde.firecloud.fixture.UserFixtures
+import org.broadinstitute.dsde.firecloud.page.workspaces.RequestAccessModal
+import org.broadinstitute.dsde.firecloud.page.workspaces.summary.WorkspaceSummaryPage
+import org.broadinstitute.dsde.workbench.auth.AuthToken
+import org.broadinstitute.dsde.workbench.config.{Config, Credentials, UserPool}
+import org.broadinstitute.dsde.workbench.fixture.{BillingFixtures, GroupFixtures, WorkspaceFixtures}
+import org.broadinstitute.dsde.workbench.service.test.{CleanUp, WebBrowserSpec}
+import org.scalatest._
+
+abstract class AuthDomainBase extends FreeSpec with Matchers
+  with CleanUp with WebBrowserSpec with WorkspaceFixtures
+  with BillingFixtures with GroupFixtures
+  with UserFixtures {
+
+  val projectName: String = Config.Projects.common
+  val defaultCuratorUser: Credentials = UserPool.chooseCurator
+  val defaultCuratorAuthToken: AuthToken = defaultCuratorUser.makeAuthToken()
+
+  protected def checkWorkspaceFailure(workspaceSummaryPage: WorkspaceSummaryPage, workspaceName: String): Unit = {
+    val error = workspaceSummaryPage.readError()
+    error should include(projectName)
+    error should include(workspaceName)
+    error should include("does not exist")
+  }
+
+  protected def checkWorkspaceFailure(workspaceSummaryPage: WorkspaceSummaryPage, projectName: String, workspaceName: String): Unit = {
+    val error = workspaceSummaryPage.readError()
+    error should include(projectName)
+    error should include(workspaceName)
+    error should include("does not exist")
+  }
```

Comment:
You could combine these two via an optional parameter with default:
```scala
protected def checkWorkspaceFailure(workspaceSummaryPage: WorkspaceSummaryPage, workspaceName: String, projectName: String = Config.Projects.common): Unit = {
  val error = workspaceSummaryPage.readError()
  error should include(projectName)
  error should include(workspaceName)
  error should include("does not exist")
}
```

Reply:
+1

Let's also be sure not to use duplicate variable names, to avoid confusion.